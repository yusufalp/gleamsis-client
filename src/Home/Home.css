.home-pic {
  width: 100%;
  max-height: 270px;
  max-width: 300px;
}

.banner {
  width: 100%;
}

.testimonials {
  margin: 90px auto;
}

blockquote {
  width: 80%;
  margin: auto;
}

blockquote .quote::before {
  content: '\201C';
}

blockquote .quote::after {
  content: '\201D';
}