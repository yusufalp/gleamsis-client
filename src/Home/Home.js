import React from 'react';

class Home extends React.Component {
  render() {
    return (
      <div>
        <header>
          <h1>GleamSIS</h1>
          <p>
            Welcome to GleamSIS. Your one stop student information system.
          </p>
        </header>
        <main className="group">
          <section className="item">
            <h2>All you need is in one place</h2>
            <p>
              You can search for your teachers, students and courses
              in one place.
            </p>
          </section>
          <section className="item">
            <h2>Access your data with ease</h2>
            <p>
              With visual data display, you can easily see your Gleam usage.
            </p>
          </section>
          <section className="item">
            <h2>Provide your staff great experience</h2>
            <p>
              Improve your staff's performance and let them manage their time effectively
            </p>
          </section>
        </main>
      </div>
    )
  }
}

export default Home;