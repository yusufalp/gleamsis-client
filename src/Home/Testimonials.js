import React from 'react';

class Testimonials extends React.Component {
  render() {
    let testimonials = [
      {
        quote: 'Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.',
        author: '—Aldous Huxley,',
        cite: 'Brave New World'
      },
    ]
    return (
      <div>
        {testimonials.map((testimonial, i) =>
          <blockquote key={i} cite="https://www.huxley.net/bnw/four.html">
            <p className='quote'>{testimonial.quote}</p>
            <p>{testimonial.author} <cite>{testimonial.cite}</cite></p>
          </blockquote>
        )}
      </div>
    )
  }
}

export default Testimonials;