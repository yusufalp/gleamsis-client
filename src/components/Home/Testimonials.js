import React from 'react';

class Testimonials extends React.Component {
  render() {
    let testimonials = [
      {
        quote: 'GleamSIS has saved us more than half of the time we used to spend organizing data about our students, parents and teachers. Now with a few clicks we can access it right away. User friendly, practical and just exactly what we need.',
        author: 'â€”Sevde Meva Alp,',
        cite: 'Principal at That School'
      },
    ];
    return (
      <div>
        {testimonials.map((testimonial, i) =>
          <blockquote key={i} cite='https://www.huxley.net/bnw/four.html'>
            <p className='quote'>{testimonial.quote}</p>
            <p>{testimonial.author} <cite>{testimonial.cite}</cite></p>
          </blockquote>
        )}
      </div>
    );
  }
}

export default Testimonials;