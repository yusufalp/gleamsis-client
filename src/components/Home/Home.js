import React from 'react';
import { Link } from 'react-router-dom';
import './Home.css';
import Testimonials from './Testimonials';

class Home extends React.Component {
  render() {
    return (
      <div>
        <header>
          <h1>GleamSIS</h1>
          <p>
            Welcome to GleamSIS. Your one stop student information system.
          </p>
        </header>
        <main>
          <section className='group'>
            <div className='item'>
              <h2>All you need is in one place</h2>
              <p>
                You can use information for your teachers, students and courses
                in one place. Every school generate thousands of data every single day.
                Keeping track of the data, accessing it and using it every day is made simple with
                GleamSIS.
            </p>
            </div>
            <div className='item'>
              <img className='home-pic' src='images/gleam3.png' alt=''></img>
            </div>
          </section>
          <section className='group2'>
            <div className='item'>
              <h2>Access your data with ease</h2>
              <p>
                With visual data display, you can use your time better. Filter and sort
                through what you need, have GleamSIS generate an awesome looking graph, share it
                with others and let the knowledge flow.
            </p>
            </div>
            <div className='item'>
              <img className='home-pic' src='images/gleam2.png' alt=''></img>
            </div>
          </section>
          <section className='group'>
            <div className='item'>
              <h2>Provide your staff great experience</h2>
              <p>
                Improve your staff's performance and let them manage their time effectively. You can
                see their usage time, offer help and support, or suggest curated tips and tricks that will
                improve their performance.
            </p>
            </div>
            <div className='item'>
              <img className='home-pic' src='images/gleam4.png' alt=''></img>
            </div>
          </section>
          <img className='banner' src='images/banner3.png' alt=''></img>
          <section>
            <h2 className='section-title'>Want to learn more</h2>
            <Link to='/contact' className='btn-add'>Contact Us</Link>
          </section>
          <section className='testimonials'>
            <h2 className='section-title'>Testimonials</h2>
            <Testimonials />
          </section>
        </main>
      </div>
    )
  }
}

export default Home;