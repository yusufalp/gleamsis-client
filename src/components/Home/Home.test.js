import React from 'react';
import ReactDOM from 'react-dom';
import Home from './Home';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

describe('Home component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <MemoryRouter>
        <Home />
      </MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Home />
        </MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});