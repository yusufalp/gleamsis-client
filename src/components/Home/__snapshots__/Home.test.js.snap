// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders the UI as expected 1`] = `
<div>
  <header>
    <h1>
      GleamSIS
    </h1>
    <p>
      Welcome to GleamSIS. Your one stop student information system.
    </p>
  </header>
  <main>
    <section
      className="group"
    >
      <div
        className="item"
      >
        <h2>
          All you need is in one place
        </h2>
        <p>
          You can use information for your teachers, students and courses in one place. Every school generate thousands of data every single day. Keeping track of the data, accessing it and using it every day is made simple with GleamSIS.
        </p>
      </div>
      <div
        className="item"
      >
        <img
          alt=""
          className="home-pic"
          src="images/gleam3.png"
        />
      </div>
    </section>
    <section
      className="group2"
    >
      <div
        className="item"
      >
        <h2>
          Access your data with ease
        </h2>
        <p>
          With visual data display, you can use your time better. Filter and sort through what you need, have GleamSIS generate an awesome looking graph, share it with others and let the knowledge flow.
        </p>
      </div>
      <div
        className="item"
      >
        <img
          alt=""
          className="home-pic"
          src="images/gleam2.png"
        />
      </div>
    </section>
    <section
      className="group"
    >
      <div
        className="item"
      >
        <h2>
          Provide your staff great experience
        </h2>
        <p>
          Improve your staff's performance and let them manage their time effectively. You can see their usage time, offer help and support, or suggest curated tips and tricks that will improve their performance.
        </p>
      </div>
      <div
        className="item"
      >
        <img
          alt=""
          className="home-pic"
          src="images/gleam4.png"
        />
      </div>
    </section>
    <img
      alt=""
      className="banner"
      src="images/banner3.png"
    />
    <section>
      <h2
        className="section-title"
      >
        Want to learn more
      </h2>
      <a
        className="btn-add"
        href="/contact"
        onClick={[Function]}
      >
        Contact Us
      </a>
    </section>
    <section
      className="testimonials"
    >
      <h2
        className="section-title"
      >
        Testimonials
      </h2>
      <div>
        <blockquote
          cite="https://www.huxley.net/bnw/four.html"
        >
          <p
            className="quote"
          >
            GleamSIS has saved us more than half of the time we used to spend organizing data about our students, parents and teachers. Now with a few clicks we can access it right away. User friendly, practical and just exactly what we need.
          </p>
          <p>
            â€”Sevde Meva Alp,
             
            <cite>
              Principal at That School
            </cite>
          </p>
        </blockquote>
      </div>
    </section>
  </main>
</div>
`;
